runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1582,8 +1582,8 @@ TR::ILOpCodes TR_VectorAPIExpansion::ILOpcodeFromVectorAPIOpcode(int32_t vectorA
       case VECTOR_OP_SUB: return scalar ? TR::ILOpCode::subtractOpCode(elementType) : TR::ILOpCode::createVectorOpCode(OMR::vsub, vectorType);
       case VECTOR_OP_MUL: return scalar ? TR::ILOpCode::multiplyOpCode(elementType) : TR::ILOpCode::createVectorOpCode(OMR::vmul, vectorType);
       case VECTOR_OP_DIV: return scalar ? TR::ILOpCode::divideOpCode(elementType) : TR::ILOpCode::createVectorOpCode(OMR::vdiv, vectorType);
-      case VECTOR_OP_MIN: return TR::BadILOp;
-      case VECTOR_OP_MAX: return TR::BadILOp;
+      case VECTOR_OP_MIN: return scalar ? TR::BadILOp : TR::ILOpCode::createVectorOpCode(OMR::vmin, vectorType);
```

Comment:
That's definitely something that should be added, but I don't think it's within scope here, since it's not necessary for the vectorized min/max operations. We can review scalarization opportunities in a separate PR